prime-router/metadata/fhir_mapping/hl7/datatypes/XTN/ContactPoint.yml

```diff
@@ -25,50 +25,51 @@ value_3:
 
 value_4:
   condition: $telEquipType NOT_EQUALS Internet && $telEquipType NOT_EQUALS X.400 && $unformattedTelNum NOT_NULL
-  valueOf: unformattedTelNum
+  valueOf: $unformattedTelNum
   vars:
     telEquipType: String, XTN.3
     unformattedTelNum: String, XTN.12
 
 value_5:
   condition: $telEquipType NULL && $unformattedTelNum NOT_NULL
-  valueOf: unformattedTelNum
+  valueOf: $unformattedTelNum
   vars:
     telEquipType: String, XTN.3
     unformattedTelNum: String, XTN.12
 
 #mapping from HL7 Concept Map: TelecomUseCode
 #https://docs.google.com/spreadsheets/d/1EE3a5UIGXLi8wMDr49fWTEVx8QS2GFqYgitE3vLOQTQ/edit#gid=0
 use_1:
-  condition: $useCode EQUALS PRN && $hl7Use NULL
+  condition: $hl7Use EQUALS PRN && $useCode NULL
   type: STRING
   value: "home"
   vars:
-    useCode: String, XTN.2
+    hl7Use: String, XTN.2
```

Comment:
nit: maybe this should be `xtnUse` instead of `hl7Use`?

Reply:
Yea good idea, will add that.